prime-dotnet-webapi-tests/Mocks/EnrolleeServiceMock.cs

```diff
@@ -157,6 +157,17 @@ public Task<bool> IsEnrolleeInStatusAsync(int enrolleeId, short statusCodeToChec
             return Task.FromResult(false);
         }
 
+        public Task<bool> IsEnrolleeInStatusAsync(int enrolleeId, ICollection<short> statusCodesToCheck)
```

Comment:
Rather than make a new method that accepts a collection (in addition to the method that accepts a single status code), I would recommend updating the method signature to `public Task<bool> IsEnrolleeInStatusAsync(int enrolleeId, params short[] statusCodesToCheck)`

Reply:
I was wondering if you'd say that :)  Definitely, and I can quickly refactor it's use in the EnrolleeController since that's the only place it is used

Reply:
Did as you suggested and updated the method to use `params`